.github/workflows/swift-toolchain.yml

```diff
@@ -347,12 +423,77 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - arch: 'amd64'
+          - arch: amd64
             BUILD_TOOLS: NO
-          - arch: 'arm64'
+            BUILD_DATA: YES
+            cc: cl
+            cflags: ${{ needs.context.outputs.CMAKE_C_FLAGS }}
+            cxx: cl
+            cxxflags: ${{ needs.context.outputs.CMAKE_CXX_FLAGS }}
+            os: Windows
+            extra_flags:
+
+          - arch: arm64
             BUILD_TOOLS: NO
-          - arch: 'x86'
+            BUILD_DATA: YES
+            cc: cl
+            cflags: ${{ needs.context.outputs.CMAKE_C_FLAGS }}
+            cxx: cl
+            cxxflags: ${{ needs.context.outputs.CMAKE_CXX_FLAGS }}
+            os: Windows
+            extra_flags:
+
+          - arch: x86
             BUILD_TOOLS: YES
+            BUILD_DATA: YES
+            cc: cl
+            cflags: ${{ needs.context.outputs.CMAKE_C_FLAGS }}
+            cxx: cl
+            cxxflags: ${{ needs.context.outputs.CMAKE_CXX_FLAGS }}
+            os: Windows
+            extra_flags:
+
+          - arch: arm64
+            BUILD_TOOLS: NO
+            BUILD_DATA: NO
+            cc: clang
+            cflags: -ffunction-sections -fdata-sections -g
+            cxx: clang++
+            cxxflags: -ffunction-sections -fdata-sections -g
+            os: Android
+            extra_flags: -DCMAKE_POSITION_INDEPENDENT_CODE=YES -D CMAKE_ANDROID_API=${{ needs.context.outputs.ANDROID_API_LEVEL }} -D CMAKE_ANDROID_ARCH_ABI=arm64-v8a "-DCMAKE_MAKE_PROGRAM=${{ needs.context.outputs.WINDOWS_VISUAL_STUDIO_PATH }}\Common7\IDE\CommonExtensions\Microsoft\CMake\Ninja\ninja.exe"
```

Comment:
I think that we should push down the default for `CMAKE_POSITION_INDEPENDENT_CODE` into the ICU CMakeLists.txt.

Reply:
https://github.com/swiftlang/swift-installer-scripts/pull/308